pkg/install/deploystorage.go

```diff
@@ -305,20 +296,8 @@ func (i *Installer) deployStorageTemplate(ctx context.Context, installConfig *in
 
 		// TODO: there is probably an undesirable race condition here - check if etags can help.
 
-		timeoutCtx, cancel := context.WithTimeout(ctx, 10*time.Minute)
-		defer cancel()
-
-		var s *mgmtnetwork.Subnet
-		err := wait.PollImmediateUntil(10*time.Second, func() (bool, error) {
-			s, err = i.subnet.Get(ctx, subnetID)
+		s, err := i.subnet.Get(ctx, subnetID)
 
-			if hasAuthorizationFailedError(err) {
-				i.log.Print(err)
-				return false, nil
```

Comment:
This new install step, `attachNSGsAndPatch`, will be retried from outside in the event of an authorization error.